src/Sulu/Bundle/SecurityBundle/Controller/ResettingController.php

```diff
@@ -491,7 +491,7 @@ private function generateTokenForUser(UserInterface $user)
      */
     private function generateTokenHash(string $token): string
     {
-        return \hash('sha1', $this->secret . $token);
+        return \hash('sha1', $this->secret . '%' . $token);
```

Comment:
sadly this will change the hash as the additional `%` was behind the token not in front of it. I will discuss this with @chirimoya if we do a bc break here or not.

Reply:
This shouldn't change the behavior. Before the change the percent sign was at the end of $this->secret now it's here. If we do a BC Break then we should not rely on the secret here.